tools/mo/openvino/tools/mo/front/common/partial_infer/eltwise.py

```diff
@@ -80,6 +81,34 @@ def broadcast_dims(dim1, dim2):
             node.out_port(0).data.set_value(op(node.out_node().value, values[i + 1]))
 
 
+def eltwise_reverse_infer(node: Node):
+    input_1_shape = node.in_port(0).data.get_shape()
+    input_2_shape = node.in_port(1).data.get_shape()
+    output_shape = node.out_port(0).data.get_shape()
+
+    if output_shape is not None:
+        out_rank = len(output_shape)
+        deduced_in_shape = undefined_shape_of_rank(out_rank)
+
+        if input_1_shape is not None and input_2_shape is None and out_rank > len(input_1_shape):
```

Comment:
also I think we need to check `auto_broadcast` attribute of eltwise node and based on it to have different logic. `pdpd` is not needed

Reply:
Agree. Implemented different logic depending from `auto_broadcast `.